requirements.txt

```diff
@@ -0,0 +1,17 @@
+# s3/boto, let aiobotocore decide botocore/boto3/awscli versions
+aiobotocore[boto3,awscli]
```

Comment:
we do not support latest `aibotocore`
`aiobotocore[boto3,awscli]==0.11.1` is what we have in sandbox that works